lab3/main.py

```diff
@@ -0,0 +1,71 @@
+
+def max_equal(arr, n):
+    maxLen = Len = 1
+
+    for i in range(1, len(arr)):
+        if arr[i] == arr[i-1]:
+            Len = Len + 1
+        else:
+            maxLen = max(maxLen, Len)
+            Len = 1
+
+    maxLen = max(maxLen, Len)
+    return maxLen
+
+
+
+
+def max_ascending(arr, n):
+    maxLen = Len = 1
+
+    for i in range(1, len(arr)):
+        if arr[i] > arr[i-1]:
+            Len = Len + 1
+        else:
+            maxLen = max(maxLen, Len)
+            Len = 1
+    maxLen = max(maxLen, Len)
+    return maxLen
+
+
+
+
+if __name__ == '__main__':
+
+    menu = {}
+    menu['1'] = "Citirea unei liste de numere intregi"
+    menu['2'] = " Gasirea secventelor de lungime maxima - toate elementele egale"
+    menu['3'] = " Gasirea secventelor de lungime maxima - in ordine crescatoare"
+    menu['4'] = "Iesire"
+
+    while True:
+        for options in menu:
+            print(options, end=" ")
+            print(menu[options])
+
+        selected = int(input("Introduceti un numar:"))
```

Comment:
until now it was ok, but from now it's not really nice to see your program crashing when providing a bad value

it should not crash if you provide a string when you expect an int